Makefile-os

```diff
@@ -52,35 +17,13 @@ update_docker:
 	docker-compose restart web
 	docker-compose restart worker
 
-.PHONY: initialize_docker
-initialize_docker:
-	docker-compose exec web make initialize
-
-.PHONY: debug
-debug:
-	docker-compose exec web supervisorctl fg olympia
-
 .PHONY: shell
 shell:
 	docker-compose exec web bash
 
-.PHONY: setup-ui-tests
-setup-ui-tests:
-	docker-compose exec web make setup-ui-tests
-
-.PHONY: djshell
-djshell:
-	docker-compose exec web ./manage.py shell_plus
-
-.PHONY: dbshell
-dbshell:
-	docker-compose exec web ./manage.py dbshell
-
-.PHONY: reload-uwsgi
-reload-uwsgi:
-	docker-compose exec web uwsgi --reload /code/docker/artifacts/addons-server-uwsgi-master.pid
+%:
+	# This directs any other recipe (command) to the web container's make.
+	docker-compose exec web make $(MAKECMDGOALS) ARGS=$(ARGS)
```

Comment:
For permissions, I wonder if this should be specified as the `olympia` user see also https://github.com/mozilla/addons-server/pull/12153

Reply:
We could.  I've never found it necessary locally when running the commands but I don't suppose it hurts.

Reply:
Are there any downsides to this?  I.e. sometimes we might not _want_ commands executed as `olympia`?

According to the docker docs we can specify a default user with `USER olympia` in the Dockerfile so perhaps that's an alternative so we don't need to specify on each docker-compose command.

Reply:
I think the main thing to think about is that if it's not `olympia` it'll be `root` for the case where the make file is running on the container, which seems to have been the source of perms issues. That might not make sense for every command but then lower privileges probably won't hurt, but we'll need to test it out.

Reply:
I added `USER olympia` to the Dockerfile instead.  https://github.com/mozilla/addons-server/pull/12121/commits/208fcb712ab1d45d06bb8f66256ce8b7c927e22b